src/resolver.rs

```diff
@@ -64,9 +94,19 @@ impl ResolverMap {
         let entries = maps::parse(pid, maps::is_symbolization_relevant)?;
 
         for entry in entries {
-            let backend = elf_cache.find(&entry.path)?;
-            let resolver = ElfResolver::new(&entry.path, entry.range.start, backend)?;
-            let () = resolvers.push((resolver.get_address_range(), Box::new(resolver)));
+            let extension = entry.path.extension().unwrap_or_else(|| OsStr::new(""));
+            log::trace!("processing proc maps entry: {entry:#x?}");
+            if extension == OsStr::new("apk") || extension == OsStr::new("zip") {
```

Comment:
do we really have ".zip" entries on Android?

Reply:
Unlikely, but it's not wrong. If there is one that's how it would be handled. We need it for testing.